python2verilog/ir/graph.py

```diff
@@ -263,25 +281,13 @@ def get_optimal_children(self):
         assert self._optimal_true_edge and self._optimal_false_edge
         return [self._optimal_true_edge, self._optimal_false_edge]
 
-    def traverse_condition_vars(self) -> Iterator[expr.Var]:
-        """
-        Yields variables of the if conditional, doing a DPS
-        """
-
-        def rec(exp: expr.Expression):
-            if isinstance(exp, expr.UBinOp):
-                yield from rec(exp.left)
-                yield from rec(exp.right)
-            elif isinstance(exp, expr.UnaryOp):
-                yield from rec(exp.expr)
-            elif isinstance(exp, expr.Var):
-                yield exp
-            elif isinstance(exp, (expr.UInt, expr.Int)):
-                pass
-            else:
-                raise RuntimeError(f"{type(exp)}")
+    def variables(self):
```

Comment:
Include that this function returns an iterator

Reply:
I think all linting tools use the parent's type hints and docs.